src/style.css

```
@@ -1 +1,21 @@
 /* styles go here */
+.form-field {
+  display: block;
+  margin: 10px 0;
+}
+
+.fieldset {
+  margin: 0 2px 20px;
```

You don't need horizontall indents in this task
```suggestion
  margin: 0 20px;
```